client/src/main/scala/com/tripPlanner/webapp/pages/ProfileJsImpl.scala

```diff
@@ -30,43 +35,52 @@ object ProfileJsImpl extends ProfileJs {
+          })("×"),
           i(cls := "fa fa-coffee")(),
-          "This is an ", strong(".alert"), ". Use this to show important messages to the user."
+          strong("Error!"), " profile was ", strong("not"), " saved successfully"
         ),
         h3("Personal Info"),
         form(cls := "form-horizontal", role := "form")(
           div(cls := "form-group")(
             label(cls := "col-lg-3 control-label")("First name:"),
             div(cls := "col-lg-8")(
-              input(cls := "form-control", `type` := "text", value := "Andy")
+              input(id := "firstName", name := "firstName", cls := "form-control", `type` := "text")
             )
           ),
           div(cls := "form-group")(
             label(cls := "col-lg-3 control-label")("Last name:"),
             div(cls := "col-lg-8")(
-              input(cls := "form-control", `type` := "text", value := "123")
+              input(id := "lastName", name := "lastName", cls := "form-control", `type` := "text")
             )
           ),
           div(cls := "form-group")(
             label(cls := "col-lg-3 control-label")("Company:"),
             div(cls := "col-lg-8")(
-              input(cls := "form-control", `type` := "text", value := "")
+              input(id := "company", name := "company", cls := "form-control", `type` := "text")
             )
           ),
           div(cls := "form-group")(
             label(cls := "col-lg-3 control-label")("Time Zone:"),
             div(cls := "col-lg-8")(
               div(cls := "ui-select")(
-                select(id := "user_time_zone", cls := "form-control")(
+                select(id := "userTimezone", name := "userTimezone", cls := "form-control")(
                   option(value := "Hawaii")("(GMT-10:00) Hawaii"),
                   option(value := "Alaska")("(GMT-09:00) Alaska"),
-                  option(value := "Pacific Time (US &amp; Canada)")("(GMT-08:00) Pacific Time (US &amp; Canada)"),
+                  option(value := "Pacific Time (US & Canada)")("(GMT-08:00) Pacific Time (US & Canada)"),
                   option(value := "Arizona")("(GMT-07:00) Arizona"),
-                  option(value := "Mountain Time (US &amp; Canada)")("(GMT-07:00) Mountain Time (US &amp; Canada)"),
-                  option(value := "Central Time (US &amp; Canada)")("(GMT-06:00) Central Time (US &amp; Canada)"),
-                  option(value := "Eastern Time (US &amp; Canada)", selected := "selected")("(GMT-05:00) Eastern Time (US &amp; Canada)"),
+                  option(value := "Mountain Time (US & Canada)")("(GMT-07:00) Mountain Time (US & Canada)"),
+                  option(value := "Central Time (US & Canada)")("(GMT-06:00) Central Time (US & Canada)"),
+                  option(value := "Eastern Time (US & Canada)", selected := "selected")("(GMT-05:00) Eastern Time (US & Canada)"),
                   option(value := "Indiana (East)")("(GMT-05:00) Indiana (East)")
```

Comment:
Should add a technical task to populate this list from the db or at least the server side in the future